src/main/resources/application.yml

```diff
@@ -23,6 +23,16 @@ spring:
       resourceserver:
         jwt:
           jwk-set-uri: ${hmpps.auth.url}/.well-known/jwks.json
+      client:
+        registration:
+          prisons-api:
+            provider: hmpps-auth
+            client-id: approved-premises-api
```

Comment:
This needs to match local HMPPS Auth credentials defined [here](https://github.com/ministryofjustice/hmpps-auth/blob/f2c661d8e7237e8c500682366ad87e0a0b0b6359/src/main/resources/db/dev/data/auth/V900_0__clients.sql#L34) and should be `hmpps-accredited-programmes-client-1` (`approved-premises` is another project, the one we've taken all of this from!)